spec/integration/authorization/pundit_spec.rb

```diff
@@ -0,0 +1,228 @@
+require 'spec_helper'
+
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def new?
+    user.roles.include? :admin
+  end
+
+  def show?
+    true
+  end
+
+  def update?
+    user.roles.include? :admin
+  end
+
+  def create?
+    user.roles.include? :admin
+  end
+
+  def edit?
+    user.roles.include? :admin
+  end
+
+  def destroy?
+    user.roles.include? :admin
+  end
+
+  def rails_admin?(action)
```

Comment:
I suppose that the purpose of this method is to enable different authorization for front-app and RailsAdmin backend, but that's not necessary. Just checking `create?`, `update?`... also in RailsAdmin side is enough.

If users need such authorization, they can customize [action configuration](https://github.com/sferik/rails_admin/wiki/Actions#through-authorization) like:
```ruby
config.actions do
  new do
    authorization_key :rails_admin_new
  end
  edit do
    authorization_key :rails_admin_edit
  end
  ...
end
```
In this way, RA will check for `rails_admin_new?` and `rails_admin_edit?`, instead of `new?` and `edit?`.